includes/classes/blocks/class-bindings.php

```
@@ -960,4 +968,92 @@ public function render_permalink_block( $block_content, $parsed_block, $block_ob
 		return $block_content;
 	}
+
+	/**
+	 * Filter the cover block content
+	 *
+	 * @param string $block_content The block content about to be rendered
+	 * @param array  $parsed_block         The full block, including name and attributes
+	 * @param object $instance      The block instance
+	 * @return string
+	 */
+	public function render_banner_block( $block_content, $parsed_block, $instance ) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$instance'. (Unused Code Rules)

(UnusedFormalParameter)